app/api/log.v2/infrastructure/Logger.ts

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable max-classes-per-file */
+import { getTenant } from 'api/common.v2/database/getConnectionForCurrentTenant';
+import { Tenant } from 'api/tenants/tenantContext';
+import {
+  LogEntryInterface,
+  LogLevelInterface,
+  LoggerInterface,
+} from '../contracts/LoggerInterfaces';
+
+class LogLevel implements LogLevelInterface {
+  name: string;
+
+  severity: number;
+
+  constructor(name: string, severity: number) {
+    this.name = name;
+    this.severity = severity;
+  }
+}
+
+const LogLevels: Record<string, LogLevel> = {
+  DEBUG: new LogLevel('DEBUG', 10),
```

Comment:
If there's going to always be a single application-wise instance of each log level I think it makes more sense to just have plain objects, in which case an interface makes sense for typing purposes.

Reply:
Sounds good!